package.json

```diff
@@ -26,16 +26,16 @@
     "@preact/compat": "0.0.4",
     "@promster/express": "5.0.3",
     "@promster/server": "6.0.2",
-    "@reach/tooltip": "0.15.2",
+    "@reach/tooltip": "0.15.3",
     "@react-aria/interactions": "3.5.0",
     "@react-spectrum/utils": "3.6.1",
     "@react-stately/toggle": "3.2.2",
-    "@stacks/auth": "2.0.0-beta.1",
-    "@stacks/blockchain-api-client": "0.62.1",
+    "@stacks/auth": "2.0.1-beta.1",
+    "@stacks/blockchain-api-client": "0.63.0-beta.0",
```

Comment:
The beta version on this can be removed, correct?

Reply:
that is correct